x/evm/ante/ctx_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package ante_test
+
+import (
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	testkeyring "github.com/evmos/evmos/v16/testutil/integration/evmos/keyring"
+	"github.com/evmos/evmos/v16/testutil/integration/evmos/network"
+	evmante "github.com/evmos/evmos/v16/x/evm/ante"
+)
+
+func (suite *EvmAnteTestSuite) TestBuildEvmExecutionCtx() {
+	keyring := testkeyring.New(1)
+	network := network.New(
+		network.WithPreFundedAccounts(keyring.GetAllAccAddrs()...),
+	)
```

Comment:
Q: Is this actually necessary? Shouldn't the following be enough for this test case?
```suggestion
	network := network.New()
```